sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -401,6 +401,70 @@ public final class StorageBlobClient: PipelineClient {
         }
     }
 
+    /// Delete a blob within a storage container.
+    /// - Parameters:
+    ///   - blob: The blob name to delete.
+    ///   - container: The container name containing the blob to delete.
+    ///   - options: A `DeleteBlobOptions` object to control the delete operation.
+    ///   - completion: A completion handler that receives a `PagedCollection` of `BlobItem` objects on success.
+    public func delete(
+        blob: String,
+        inContainer container: String,
+        withOptions options: DeleteBlobOptions? = nil,
+        then completion: @escaping HTTPResultHandler<Data?>
```

Comment:
Guidelines say this should be called `completionHandler`. It probably doesn't need separate internal & external names.

Reply:
This was essentially copy-pasta. I could:

1) update it here now in this one place
2) update it here everywhere
3) leave as is here and have it updated in a different PR

Reply:
I'm fine with #1 - we can go back and update the others later, but no need to add new code now that we know we need to change later.